verification/verify.go

```diff
@@ -184,7 +184,7 @@ func IsRetryable(err error) bool {
 }
 
 func (sv *SegmentVerifier) sigVerification(params *Params) error {
-	if params.Orchestrator == nil || params.Orchestrator.TicketParams == nil {
+	if params.Orchestrator == nil || params.Orchestrator.Address == nil {
```

Comment:
Should it actually exit early in both cases, like this?

```go
if params.Orchestrator == nil || params.Orchestrator.Address == nil  || params.Orchestrator.TicketParams == nil {
    return nil
}
```

Otherwise, I guess we can get panic in the next lines.

Reply:
Yes, this is better. I will update.